css/main.css

```
@@ -0,0 +1,114 @@
+.top-bottom-section {
+  height: 540px;
+  background: #cfe5ff;
+}
+
+.main-section {
+  background-color: #fcfcfc;
```

💊 제안
하나의 레포에는 동일한 규칙으로 코드가 작성되는 것이 좋습니다~ 지금의 경우, 둘다 배경색을 지정하는 것이니 background-color 사용을 추천드려요!


```suggestion
  background-color: #cfe5ff;
}

.main-section {
  background-color: #fcfcfc;
```